presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDirectSelectiveStreamReader.java

```diff
@@ -306,15 +306,19 @@ else if (nullsAllowed) {
     private void skip(int items)
             throws IOException
     {
-        if (dataStream == null) {
+        // in case of an empty varbinary both the presentStream and dataStream are null and only lengthStream is present.
+        if (dataStream == null && presentStream != null) {
             presentStream.skip(items);
         }
         else if (presentStream != null) {
             int lengthToSkip = presentStream.countBitsSet(items);
             dataStream.skip(lengthStream.sum(lengthToSkip));
         }
         else {
```

Comment:
nit: 
```java
else if (dataStream != null) {
    dataStream.skip(lengthStream.sum(items));
}

Reply:
@highker  in that case lengthStream will not be proceed right

Reply:
I'm not sure actually. The snippet I put should be semantically equivalent as yours, isn't it?

Reply:
lengthStream.sum advances the currentPosition.
lets say we need to skip 10 positions then lengthStream.sum(10) will return the number of data items to skip and at the same time skips 10 positions in the lengthStream, but the snippet you have does not enter inside the codition as dataStream is null.

Reply:
Got it. I didn't see `sum` has side effect.